packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -150,13 +172,15 @@ const Autocomplete = <
   isReadOnly,
   hint,
   label,
+  ListboxComponent,
```

Comment:
Why are we allowing passing in the `ListboxComponent` itself?

That would allow devs to bypass Odyssey's pre-defined set of Autocomplete styles to be something that doesn't match the design system.